img/s/shy.gg.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="2048" height="2048">
```

Comment:
```suggestion
```
Apologies I didn't select the whole file when I made the previous suggestion, this part can be removed as it's not needed.